src/maintainer/maintainer_faq.rst

```diff
@@ -181,3 +181,15 @@ FAQ
   .. note::
 
     Even if the maintainer isn't active anymore, we generally like to keep them in the list of maintainers and not remove them, in case they want to take up maintenance at a later date.
+
+.. _mfaq_changes_to_major_projects:
+
+:ref:`(Q) <mfaq_changes_to_major_projects>` **Does ``conda-forge`` ever make significant changes or apply code patches to significant upstream packages?**
+
+  We generally do not make changes, but there are many notable exceptions and we have no set policy. These changes currently fall into
```

Comment:
```suggestion
  We generally try to avoid changes, but there are many notable exceptions and we have no set policy. These changes currently fall into
```